org.eclipse.paho.android.service/build.gradle

```diff
@@ -42,7 +42,8 @@ configurations {
 
 dependencies {
     implementation "com.android.support:support-v4:${rootProject.ext.supportLibVersion}"
-    implementation "in.mohalla.paho:in.mohalla.paho.client.mqttv3:1.2.7.timer-ping-sender"
+    //implementation "in.mohalla.paho:in.mohalla.paho.client.mqttv3:1.2.7.timer-ping-sender"
```

Comment:
TODO: revert this change and delete the jar file once https://github.com/ShareChat/paho.mqtt.java/pull/4 is done

Reply:
Done